mantidproject/mantid
# Refactor Save Options Handling

**Description of work.**
Refactors the Save Options handling in the ISIS SANS GUI to correctly follow our GUI standards.
This is prep work for #31929 

**To test:**
- Download the ISIS Training Data and ensure the loqdemo folder is present
- Open the ISIS SANS GUI
- Set the mask file to the .TOML file in the loqdemo folder
- Set the batch file to the .csv from the same folder
- Under manage user directories change the default save dir to somewhere clean
- Set the output mode to file 
- Mess around with the tick options and check they work. Note the default options missing is expected, as it's being done in an upcoming PR

Part of #31929 

*This does not require release notes* because this partially completes the issue



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
